lib/Crypto/cryptomanager.h

```diff
@@ -313,6 +313,19 @@ class SAILFISH_CRYPTO_API CryptoManager : public QObject
     Q_DECLARE_FLAGS(Operations, Operation)
     Q_FLAG(Operations)
 
+    enum VerificationResult {
+        VerificationSucceeded,
+        VerificationFailed,
+        VerificationInvalidSignature,
+        VerificationValidSignatureExpired,
```

Comment:
can a signature expire?

Reply:
If certificate expires then I'd assume that signature should expire as well?

Reply:
How is a signature related to a ceriticate?

Reply:
In S/MIME protocol, signatures are validated by certificates for instance.

Reply:
Signatures can be valid but expired according to  https://www.gnupg.org/documentation/manuals/gpgme/Verify.html GPG_ERR_SIG_EXPIRED

Reply:
Our API currently doesn't deal with certificates in any way, as we didn't have time to mature that set of features.  Well, I guess that someone could define a signature format which included some metadata like expiry date, separately from certificates, so we can leave this here for now anyway.